src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1252,6 +1257,21 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
                     _TryStopAutoScroll(ptr.PointerId());
                 }
             }
+            //else
+            //{
+            //    // We have moved to a new position, start/reset the hover timer
+            //    _hoverTimer.Start();
+            //    _hoverPos = point.Position();
+            //}

```

Comment:
dead code

Reply:
Yep! So right now I have implemented two ways of doing this (not sure which way we want to do it, thats why both versions are in there right now and I've made this a draft PR)
1:
- Have a hoverTimer that is initialized with TermControl (but we don't start it then)
- In pointerMovedHandler, we start it (so it'll get reset everytime the pointer moves)
- When the timer fires, we check if the hovered position is a link, and if so, move the border/update the tooltip content

2:
- No hoverTimer (since the in-built xaml tooltip deals with the hovering for us)
- In pointerMovedHandler, check if the new position is a link and if so, move the border/update the tooltip content

Number 2 seems much more straightforward but I'm not sure about it because it requires checking whether the new position is a hyperlink _everytime_ we move the pointer, as opposed to only checking it when the hoverTimer fires (tho we do save on not needing to reset the timer everytime the pointer moves)

Reply:
Resolved, we are going with number 2.